### DISCUSSION TITLE
How can two job intervals run, or the second job can delay startup?

### DISCUSSION THREAD

**Discussion by piqihaha** [swar/Swar-Chia-Plot-Manager#117] [Q&A]
I have two Intel P3600 1.6T SSDs, two 8T HDDs, and I want to run two jobbs, each corresponding to an SSD and an HDD, (SSD HDD runs concurrently), which can be started in two robs in turn, the first plot belongs to A1, the second plot belongs to plotA2, the third plot belongs to A1, and the fourth belongs to plotA2 ... Or, job-A1's four plots all started, and then start job-A2 plot, I won't configure, who can help me

global:
max_ concurrent: 8
jobs:

name: A1
max_plots: 999
farmer_public_key:
pool_public_key:
temporary_directory: E:\ #（this is SSD 1.4T）
temporary2_directory:
destination_directory: H:\ #（this is HDD 8T）
size: 32
bitfield: true
threads: 2
buckets: 128
memory_buffer: 4096
max_concurrent: 4
max_concurrent_with_start_early: 7
stagger_minutes: 10
max_for_phase_1: 2
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 0
temporary2_destination_sync: false

name: A2
max_plots: 999
farmer_public_key:
pool_public_key:
temporary_directory: F:\ #（this is SSD 1.4T）
temporary2_directory:
destination_directory: G:\ #（this is HDD 8T）
size: 32
bitfield: true
threads: 2
buckets: 128
memory_buffer: 4096
max_concurrent: 4
max_concurrent_with_start_early: 3
stagger_minutes: 15
max_for_phase_1: 2
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 0
temporary2_destination_sync: false

**Comment by swar** [comment]
Please use Discussions if this is not pertaining to a software issue.

**Reply by jonathanhatch** [reply]
Wait. Isn't this discussions?

**Reply by Mareknown** [reply]
Did you find out answer to your question? Just want to learn how to delay the startup on the second job and see an example config for two jobs. Anyone can share please...
